js/build.js

```diff
@@ -0,0 +1,3 @@
+Fliplet.Widget.instance('collapse-end', function() {
```

Comment:
Can you confirm why this is necessary? Accordions are initialised using [this file](https://github.com/Fliplet/fliplet-widget-accordion-start/pull/28/files/787b08c1e5d839de413fbab265af1a992a471c7c..f0b72e1484d74bedea33c498e3f55e6339803c83#diff-a6a69823860696351be715a6f34f14d877f3d832c332b625e19cf4382b03d75a) from the https://github.com/Fliplet/fliplet-widget-accordion-start component. The accordion end component shouldn't need to do anything to support localisation.